ConvivaTestApp/src/androidTest/java/com/bitmovin/analytics/conviva/testapp/PlaybackErrorTrackingTests.kt

```diff
@@ -17,8 +17,8 @@ class PlaybackErrorTrackingTests: TestBase() {
         // load invalid asset without initializing Conviva session explicitly
         activityScenario.onActivity { activity: MainActivity ->
             activity.bitmovinPlayer.load(INVALID_DASH_VOD_SOURCE)
-            Thread.sleep(2000)
         }
+        Thread.sleep(2000)
```

Comment:
Same question as in `PlaybackControlsTrackingTests.kt` around the `Thread.sleep`.